Libraries/NavigationExperimental/NavigationExperimental.js

```diff
@@ -41,6 +43,8 @@ const NavigationExperimental = {
   Card: NavigationCard,
   CardStack: NavigationCardStack,
   Header: NavigationHeader,
+  HeaderTitle: NavigationHeaderTitle,
+  HeaderBackButton: NavigationHeaderBackButton,
```

Comment:
Do we need to export these individually? Instead lets export them as a part of the header component